diff --git a/android/build.gradle b/android/build.gradle
index 5400528c91eff59ed7b480bc10cf1688f4326f14..ac1fb10f67651c53b55a068efba3eec351e23b20 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -7,6 +7,7 @@ def DEFAULT_TARGET_SDK_VERSION = 26
 def DEFAULT_MIN_SDK_VERSION = 16
 
 android {
+    namespace "io.xogus.reactnative.versioncheck"
     compileSdkVersion rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : DEFAULT_COMPILE_SDK_VERSION
     buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
 
diff --git a/android/src/main/AndroidManifest.xml b/android/src/main/AndroidManifest.xml
index 2d04cf430efbda59f41f65bfc0d33c8b6606578a..e07461ce3c69529d6c344ccb53ffe85ee9bfc5b7 100644
--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -1,6 +1,5 @@
 
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="io.xogus.reactnative.versioncheck">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
 </manifest>
   
